swupd/packs_test.go

```diff
@@ -385,8 +409,12 @@ func mustValidateZeroPack(t *testing.T, manifestPath, packPath string) {
 		if err != nil {
 			t.Fatalf("error reading pack: %s", err)
 		}
-		if hdr.Name == "staged/" || !strings.HasPrefix(hdr.Name, "staged/") {
-			continue
+		if hdr.Name == "staged/" || hdr.Name == "delta/" {
```

Comment:
Another good place to put the comment that you've already encountered these.

Reply:
The error message talks about "multiple %s found", so I think we are covered with that plus the other two comments added.